src/components/Footer.tsx

```diff
@@ -1,105 +1,22 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  Icon,
+  List,
+  ListItem,
+  SimpleGrid,
+  useToken,
+} from "@chakra-ui/react"
+import { graphql, StaticQuery } from "gatsby"
 import React from "react"
-import styled from "@emotion/styled"
+import { FaGithub, FaTwitter, FaYoutube, FaDiscord } from "react-icons/fa"
 import { useIntl } from "react-intl"
-import { StaticQuery, graphql } from "gatsby"
-import { Icon } from "@chakra-ui/react"
-import { FaDiscord, FaGithub, FaTwitter, FaYoutube } from "react-icons/fa"
-
+import { Lang } from "../utils/languages"
 import { getLocaleTimestamp } from "../utils/time"
-import Translation from "./Translation"
-import Link from "./Link"
 import { isLangRightToLeft, TranslationKey } from "../utils/translations"
-import { Lang } from "../utils/languages"
-
-const StyledFooter = styled.footer`
-  padding-top: 3rem;
-  padding-bottom: 4rem;
-  padding: 1rem 2rem;
-`
-
-const FooterTop = styled.div`
-  font-size: ${(props) => props.theme.fontSizes.s};
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  flex-wrap: wrap;
-`
-
-const LastUpdated = styled.div`
-  color: ${(props) => props.theme.colors.text200};
-`
-
-const LinkGrid = styled.div`
-  display: grid;
-  grid-template-columns: repeat(6, auto);
-  grid-gap: 1rem;
-  justify-content: space-between;
-  @media (max-width: 1300px) {
-    grid-template-columns: repeat(3, auto);
-  }
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    grid-template-columns: repeat(2, auto);
-  }
-  @media (max-width: 500px) {
-    grid-template-columns: auto;
-  }
-`
-
-const LinkSection = styled.div``
-
-const SectionHeader = styled.h3`
-  font-size: 0.875rem;
-  line-height: 1.6;
-  margin: 1.14em 0;
-  font-weight: bold;
-`
-
-const List = styled.ul`
-  font-size: 0.875rem;
-  line-height: 1.6;
-  font-weight: 400;
-  margin: 0;
-  list-style-type: none;
-  list-style-image: none;
-`
-
-const ListItem = styled.li`
-  margin-bottom: 1rem;
-`
-
-const FooterLink = styled(Link)`
-  text-decoration: none;
-  color: ${(props) => props.theme.colors.text200};
-  svg {
-    fill: ${(props) => props.theme.colors.text200};
-  }
-  &:after {
-    color: ${(props) => props.theme.colors.text200};
-  }
```

Comment:
This was the culprit to the missing external icons when applied with Chakra. The object created with the `__after` prop on the `Link` component was overriding the styling applied [within the component](https://github.com/ethereum/ethereum-org-website/blob/2713f4cfdb4596f9d73ee0a9dc2a0661f5f58a0e/src/components/Link.tsx#L95-L99)

This wasn't needed anyway because the pseudo inherits color from the parent element.